Doxyfile

```diff
@@ -0,0 +1,2729 @@
+GENERATE_TREEVIEW      = YES # optional. Also works without treeview
```

Comment:
Is this the same as the doxyfiles we use internally?

Reply:
Ok, it doesn't look like it. There might be some configuration in the internal doxyfile that we might wanna keep. If this file is not fine-tuned, I would suggest just copying the project-independent part. Otherwise maybe a diff can help us :)